src/Plugin/MultiselectFilterField/StringField.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_list_pages\Plugin\MultiselectFilterField;
+
+use Drupal\oe_list_pages\MultiSelectFilterFieldPluginBase;
+
+/**
+ * Defines the string field type multiselect filter plugin.
+ *
+ * This is used as a default for string fields which don't have anything
+ * more specific.
+ *
+ * @MultiselectFilterField(
+ *   id = "string",
+ *   label = @Translation("Default string field"),
+ *   field_types = {
+ *     "string",
+ *   },
+ *   data_types = {
+ *     "string",
+ *   },
+ *   weight = 100
+ * )
+ */
+class StringField extends MultiSelectFilterFieldPluginBase {
```

Comment:
A default plugin that a Drupal string field or a search API string field can default to, worse case scenario.